gulpfile.js

```diff
@@ -46,15 +32,7 @@ gulp.task('buildQuarkusServerExt', (done) => {
   done();
 });
 
-gulp.task('buildServer', gulp.series(['buildMicroProfileServer', 'buildQuarkusServerExt']));
-
-gulp.task('buildMicroProfileExtension', (done) => {
-  cp.execSync(mvnw() + ' -pl "' + microprofileExtension + '" clean verify -DskipTests', { cwd: microprofileExtensionDir, stdio: 'inherit' });
-  gulp.src(microprofileExtensionDir + '/' + microprofileExtension + '/target/' + microprofileExtension + '-!(*sources).jar')
-    .pipe(rename(microprofileExtension + '.jar'))
-    .pipe(gulp.dest('./jars'));
-  done();
-});
+gulp.task('buildServer', gulp.series(['buildQuarkusServerExt']));
```

Comment:
I think its a bit redundant to have this. We can delete this task and rename https://github.com/redhat-developer/vscode-quarkus/blob/7b1c4a2fbdba0c6216e1d16bde7b8d0786360fc3/gulpfile.js#L24 to 
```
gulp.task('buildServer', (done) => {
```

Reply:
fixed